gekitchensdk/erd/values/laundry/drynessnew_level.py

```diff
@@ -0,0 +1,13 @@
+import enum
```

Comment:
DrynessNewLevel looks like DrynessLevel, they should probably be combined into the same type (you can then use the same converter for both ERD values)

Reply:
Another good call, updated.

Reply:
I've been mulling this over, while they both have the same items, the values differ (eg: DrynessLevel has a INVALID level = 5, while DrynessNewLevel has a INVALID value of 0, which pushes the remaining values.

I'll consolidate them into the one set of files however I believe we'll still need to have the separate converter, unless I pass an argument to the function specifying which MAP should be referenced.